examples-sdk-s3/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright 2018-2019 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>atum-examples-sdk-s3_2.11</artifactId>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>za.co.absa</groupId>
+        <artifactId>atum-parent_2.11</artifactId>
+        <version>3.2.1-SNAPSHOT</version>
+    </parent>
+
+    <dependencies>
+        <!-- Atum -->
+        <dependency>
+			<groupId>za.co.absa</groupId>
+			<artifactId>atum-sdk-s3_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+        <dependency>
+            <groupId>za.co.absa</groupId>
+            <artifactId>atum-examples_${scala.binary.version}</artifactId>
```

Comment:
Including this only for 2 utils methods seems a bit heavy-handed. Separate utils or commons?

Reply:
I actually did it sort of deliberately to demonstrate that "s3 sdk examples" dependencies only need "sdk s3 extension" on top of what regular Atum "examples" need, nothing more.

The resulting jar may be a bit bigger than if utils or commons were separated, but I prioritized simplicity for the reader. Given the fact that these jars do not actually get used as such, my perspective was that this is acceptable.

I will think about it in relation to other modules, whether `utils` or `commons` would be beneficial elsewhere, too.

Reply:
Fair enough. I see your point. Both approaches work for me